packages/twenty-server/src/engine/integrations/message-queue/message-queue.constants.ts

```diff
@@ -9,4 +9,5 @@ export enum MessageQueue {
   calendarQueue = 'calendar-queue',
   billingQueue = 'billing-queue',
   recordPositionBackfillQueue = 'record-position-backfill-queue',
+  entityEventsToDbQueue = 'entity-events-to-db-queue',
```

Comment:
Same comment as above. I think we should clearly define that those are events from the API and not DB so later when we will use real DB events it won't be confusing

Reply:
(same discussion / it says events TO db but definitely open to better naming!)